drivers/adc/ad7124/ad7124.c

```diff
@@ -47,7 +47,15 @@
 /* Error codes */
 #define INVALID_VAL -1 /* Invalid argument */
 #define COMM_ERR    -2 /* Communication error on receive */
-#define TIMEOUT     -3 /* A timeout has occured */
+#define TIMEOUT     -3 /* A timeout has occurred */
```

Comment:
nitpick: it's generally good to hold-off on changing minor/non-functional/unrelated things [spacing, typos, etc];

most of the times, they don't matter; but sometimes [just sometimes] they can be really annoying, because of how git handles merge, rebase or cherry-pick conflicts;

unfortunately, there is no clear examples i can think of now;
these things become more important when more people work on the same code, and chances of conflicts become high;

we can leave this change as-is;

the usual way [for git] is to have a commit called `ad7214: Fix typo in timeout description` or something like that